### PR TITLE
Implementing cluster CLI tests

### LINKED ISSUE #597: Write unit/functional tests for dask cluster CLI

### PR DESCRIPTION
**Pull request by venkatBala**
Resolves #597 

- [x] I have added the tests to cover my changes.
- [x] I have updated the documentation and CHANGELOG accordingly.
- [x] I have read the CONTRIBUTING document.

### CODE REVIEW

**Comment by venkatBala** [comment]
Resolves #597

File: tests/covalent_dispatcher_tests/_cli/service_test.py [hunk 1]
```diff
@@ -290,3 +290,183 @@ def test_purge(mocker):
     )
     purge_config_mock.assert_called_once()
     assert result.output == "Covalent server files have been purged.\n"
+
+
+@pytest.mark.parametrize("workers", [1, 2, 3, 4])
+def test_cluster_size(mocker, workers):
+    """
+    Test to ensure that cluster start with the expected number of workers
+    """
```
Comment (FyzHsn) [inline tests/covalent_dispatcher_tests/_cli/service_test.py:299]: (nitpick) Add blank line after docstring.

File: tests/covalent_dispatcher_tests/_cli/service_test.py [hunk 2]
```diff
@@ -290,3 +290,183 @@ def test_purge(mocker):
...
+    get_event_loop_mock.assert_called_once()
+    unparse_addr_mock.assert_called_once()
+    cluster_size_mock.assert_called_once()
+    assert int(response.output) == workers
+
+
+def test_cluster_info(mocker):
+    """
+    Test cluster info CLI
+    """
```
Comment (FyzHsn) [inline tests/covalent_dispatcher_tests/_cli/service_test.py:328]: Blank line after doc string.

File: tests/covalent_dispatcher_tests/_cli/service_test.py [hunk 3]
```diff
@@ -290,3 +290,183 @@ def test_purge(mocker):
...
+    get_event_loop_mock.assert_called_once()
+    unparse_addr_mock.assert_called_once()
+    cluster_info_cli_mock.assert_called_once()
+    click_echo_mock.assert_called_once()
+
+
+def test_cluster_status_cli(mocker):
+    """
+    Test cluster status cli
+    """
```
Comment (FyzHsn) [inline tests/covalent_dispatcher_tests/_cli/service_test.py:353]: Blank line after doc string.

File: tests/covalent_dispatcher_tests/_cli/service_test.py [hunk 4]
```diff
@@ -290,3 +290,183 @@ def test_purge(mocker):
...
+    unparse_addr_mock.assert_called_once()
+    cluster_status_cli_mock.assert_called_once()
+    click_echo_mock.assert_called_once()
+
+
+def test_cluster_address_cli(mocker):
+    """
+    Test cluster address cli
+    """
+    loop = mock.Mock()
```
Comment (FyzHsn) [inline tests/covalent_dispatcher_tests/_cli/service_test.py:379]: Blank line after doc string.

File: tests/covalent_dispatcher_tests/_cli/service_test.py [hunk 5]
```diff
@@ -290,3 +290,183 @@ def test_purge(mocker):
...
+    get_event_loop_mock.assert_called_once()
+    unparse_addr_mock.assert_called_once()
+    cluster_cli_mock.assert_called_once()
+    click_echo_mock.assert_called_once()
+
+
+def test_cluster_logs_cli(mocker):
+    """
+    Test cluster logs cli
+    """
```
Comment (FyzHsn) [inline tests/covalent_dispatcher_tests/_cli/service_test.py:403]: Blank line after docstring.

**Review by FyzHsn — APPROVED** [review]
@venkatBala LGTM! I left some nitpick comments about adding a blank line after doc-strings. That needs to be applied throughout.

**Review by scottwn — APPROVED** [review]

**Review by scottwn — DISMISSED** [review]

**Review by scottwn — APPROVED** [review]

**Pull request merged**